core/bones/password.py

```diff
@@ -69,21 +63,58 @@ class PasswordBone(StringBone):
         "core.bones.password.tooWeakMessage",
         defaultText="The entered password is too weak."
     )
+    _password_tests: List[Tuple] = [
+        (r"(?=.*[A-Z])", translate("core.bones.password.no_capital_letters",
+                                   defaultText="The password entered has no capital letters.")),
+        (r"(?=.*[a-z])", translate("core.bones.password.no_lowercase_letters",
+                                   defaultText="The password entered has no lowercase letters.")),
+        (r"(?=.*\d)", translate("core.bones.password.no_digits",
+                                defaultText="The password entered has no digits.")),
+        (r"(?=.*\W)", translate("core.bones.password.no_special_characters",
+                                defaultText="The password entered has no special characters.")),
+
+    ]
+
+    def __init__(
+        self,
+        *,
+        min_password_length: int = 8,
+        test_threshold: int = 3,
+        password_tests: List[Tuple] = None,
+        **kwargs
+    ):
+        """
+            Initializes a new Password Bone.
+
+            :param min_password_length: The minimum length of the password all passwords with a length that is
+                    smaller this will be invalid.
+
+            :param test_threshold: The minimum number of tests the password must pass.
+            :param password_tests: A list of tuple.The tuple  contains the tests and an error message if the test fails
```

Comment:
Low-hanging fruit. Multiple tuple**s**, each has **one** test and text. Additionally it's more a reason/hint than an error message.

```suggestion
            :param password_tests: A list of tuples. The tuple contains the test and a reason for the user if the test fails.
```